### DISCUSSION TITLE
What is the state of session support?

### DISCUSSION THREAD

**Discussion by Vinccool96** [nuxt/nuxt#16508] [Questions]
What is the state of session support?

In the [core modules of the roadmap](https://v3.nuxtjs.org/community/roadmap/#-core-modules), it is said that the auth module support is planned after session support. However, I can't find how far the session support is at. Do we have any news about it? Does the [nuxt/auth repository](https://github.com/nuxt/auth) hidden or just hasn't been created?

**Comment by wobsoriano** [comment]
For the meantime, you can try https://github.com/wobsoriano/h3-session

**Reply by christopher4lis** [reply]
Module looks like the perfect solution, seems to be caught up on an infinite load though after rc.10 though.

**Comment by christopher4lis** [comment]
If anyone has any solutions as to how they implemented session support into Nuxt 3, would love to hear about it here! I'm still looking but will be sure to post my solution if I find one.

**Comment by wobsoriano** [comment]
Here’s another session module for Nuxt powered by iron session!

https://github.com/wobsoriano/nuxt-iron-session
